mirascope/core/vertex/_utils/_calculate_cost.py

```diff
@@ -0,0 +1,8 @@
+"""Calculate the cost of a Vertex API call."""
+
+
+def calculate_cost(
+    input_tokens: int | float | None, output_tokens: int | float | None, model: str
+) -> float | None:
+    """Calculate the cost of a Vertex API call."""
+    return None
```

Comment:
vertex ai has pricing:
https://cloud.google.com/vertex-ai/generative-ai/pricing

Reply:
I implement it.